pkgs/top-level/python-packages.nix

```diff
@@ -9183,24 +9187,9 @@ in {
     };
   };
 
-
-  ipy = buildPythonPackage rec {
-    version = "0.74";
-    name = "ipy-${version}";
-
-    src = pkgs.fetchurl {
-      url = "mirror://pypi/I/IPy/IPy-${version}.tar.gz";
-      sha256 = "5d6abb870c25f946c45c35cf50e66155598660f2765b35cb12e36ed5223c2b89";
-    };
-
-    # error: invalid command 'test'
-    doCheck = false;
-
-    meta = {
-      description = "Class and tools for handling of IPv4 and IPv6 addresses and networks";
-      homepage = https://pypi.python.org/pypi/IPy;
-    };
-  };
+  IPy = callPackage ../development/python-modules/IPy { };
+  # Alias to remove 
+  ipy = callPackage ../development/python-modules/IPy { };
```

Comment:
What's wrong with the old name? Most attributes in Nixpkgs are lowercase.

Reply:
Don't know what's better between the real name (as a lots of packages), or all in lowercase. I think, in pypi you can have two different packages with the same name except but different capitalized letters.

Reply:
No, that's not true. The attributes are preferably the normalized names, so `ipy` in this case.
https://www.python.org/dev/peps/pep-0503/#normalized-names